bfdd/bfd_packet.c

```diff
@@ -768,6 +768,39 @@ static void cp_debug(bool mhop, struct sockaddr_any *peer,
 		   mhop ? "yes" : "no", peerstr, localstr, portstr, vrfstr);
 }
 
+static int bfd_check_auth(struct bfd_session *bfd, struct bfd_pkt *cp){
```

Comment:
Please sanitize the function:
- Use `bool` instead of `int` since we only want to know if authentication check succeeded
- `const`ify `bfd` and `cp` since they are only read